bin/queue

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Application;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\ContainerConfig;
+
+require_once './vendor/autoload.php';
```

Comment:
Could there be a problem if run utility from different places?
May be better use absoulte path via `__DIR__` (see [yii2](https://github.com/yiisoft/yii2/blob/master/framework/yii))?

Reply:
It is not necessary when using ./ it works well for commands, whether you install the package in an app or clone it.

Reply:
I try it, and this do not work correclty.

Reply:
As it fails, to reproduce it, since I use it in the app, and in the cloned directory, and it works correctly.

Reply:
![image](https://github.com/yiisoft/yii-queue/assets/525501/0ce0902b-3fa8-4bae-9eed-5eafc386b1ee)

Reply:
done.

Reply:
Don't work. Add test.

Reply:
![image](https://github.com/yiisoft/yii-queue/assets/42547589/639fe1ed-b258-44ee-82ca-9119ffc3e3c0)

![image](https://github.com/yiisoft/yii-queue/assets/42547589/687eb4e5-8c32-4b2b-93f5-1cf127a024d5)

It is a simple script, it works from the app's main directory, and from its own directory.

Reply:
Utlitiy should work from any directory. Just use absolute path.